### ISSUE TITLE
ionic cordova resources preparation problem

### ISSUE THREAD

**Issue by GiperProger** [ionic-team/ionic-cli#4370]
**Description:**
I am using ionic 3. when I try to prepare images by using `ionic cordova resources --force` command I get next error 

**HTTP Error 503: POST https://res.ionic.io/api/v1/upload**

```
<html>
	  <head>
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<meta charset="utf-8">
		<title>Application Error</title>
		<style media="screen">
		  html,body,iframe {
			margin: 0;
			padding: 0;
		  }
		  html,body {
			height: 100%;
			overflow: hidden;
		  }
		  iframe {
			width: 100%;
			height: 100%;
			border: 0;
		  }
		</style>
	  </head>
	  <body>
		<iframe src="//www.herokucdn.com/error-pages/application-error.html"></iframe>
	  </body>
	</html>
```

Your guys advise updating the platform to the latest version and close the ticket, but it's impossible for us, we have a lot of ionic 3 applications. Please return the previous solution or give another solution.

**Comment by kensodemann** [comment]
You can use version 6 of the CLI with Ionic Framework version 3 applications. They are two separate things.

**Comment by GiperProger** [comment]
@kensodemann ok, I updated my ionic CLI and tried to perform `ionic cordova resources --force` command again. It said to me that I should update my node js version(I am using 6.9.3) When I updated my node js version and performed `ionic cordova resources --force` command it said to me 

![image](https://user-images.githubusercontent.com/11026729/77909646-88e87380-72af-11ea-9fe6-13183acbb724.png)

 I tried to perform this command but got the final error 

![image](https://user-images.githubusercontent.com/11026729/77910181-80dd0380-72b0-11ea-8637-918675ef7eeb.png)

What should I do next?

**Comment by kensodemann** [comment]
If you were on Node 6, that is _ancient_ and has been out of LTS forever now. You should be sure to stick with whatever the Node team has as LTS at any given time.

There are likely a lot of dependencies within your project that need to be updated at this point. It sounds like you have not been keeping that up to date. You are making good progress, but will likely have to have a look at your deps and rebuild them.

It may be worth doing the following:
- ionic repair
- npm outdated

On the "outdated", stick with ionic-angular 3.x.

Same with the @ionic/app-scripts, though you should probably be on the latest on both. Depending how out of date you are there, though, you may have a fairly large migration just there. Here is a v3 project that builds just fine with CLI v6: https://github.com/kensodemann/ionic3-lazy-loading. Use that as a guide for versions to use for the base libraries and @ionic-native.

Your Cordova plugins (the plugins themselves, not the @ionic-native wrappers) likely also need to be updated.

Basically, it appears that you have not kept up to date on your dependencies, and this will cause you issues over time. But you should be able to clean that all up.

**Comment by kirstenfrager** [comment]
@kensodemann I seem to be getting this same error today as well. It has been working fine up until today when I am trying to build. 

It is happening in our AngularJS/Ionic 1 application.

Current environment is as follows:

**Angular Env:**

Angular CLI: 8.3.4
Node: 13.10.1
OS: darwin x64
... 

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.803.4 (cli-only)
@angular-devkit/core         8.3.4 (cli-only)
@angular-devkit/schematics   8.3.4 (cli-only)
@schematics/angular          8.3.4 (cli-only)
@schematics/update           0.803.4 (cli-only)
rxjs                         5.5.12


**Ionic Info**

Ionic:

   ionic (Ionic CLI) : 4.5.0
   Ionic Framework   : unknown
   @ionic/v1-toolkit : not installed

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 8.0.0, ios 5.1.1
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.2, cordova-plugin-ionic-webview 2.5.1, (and 12 other plugins)

System:

   ios-deploy : 1.9.4
   ios-sim    : 8.0.2
   NodeJS     : v13.10.1 (/usr/local/Cellar/node/13.10.1/bin/node)
   npm        : 6.14.3
   OS         : macOS Catalina
   Xcode      : Xcode 11.3.1 Build version 11C504

**Comment by kensodemann** [comment]
@kirstenfrager - your Ionic CLI is rather old at 4.5.0. I suggest at least going to the latest in the "ionic" line (`npm i -g ionic`), which will get you 5.4.15.

Better yet would be:

```
npm uninstall -g ionic
npm i -g @ionic/cli  
```

That will get you on 6.4.0 of the CLI, but more importantly, on to the "go forward" package. The older CLI package will not be getting updates over time.

**Comment by kirstenfrager** [comment]
@kensodemann this has worked for me. thank you for your help!

**Comment by zhaoleisam** [comment]
how can I use  ionic 6.4.0 to start a ionic3 progect

**Comment by kensodemann** [comment]
@zhaoleisam 

`ionic start my-project blank --type=ionic-angular`

This isn't really what the GH issues list is for, though. It is for bugs and feature requests, no support, so I am going to close this out.
